omero/developers/Web.txt

```diff
@@ -94,8 +108,8 @@ repository.
 
 There are 3 parts to each page: url, view and template. For example, this code
 below is for generating an HTML page of a Dataset (see screen-shot). If you
-have OMERO.web running, you can view the page under
-`http://servername.example.org/webtest/dataset/`\ `<datasetId>`.
+have OMERO.web running and webtest installed, you can view the page under
+`http://`\ `<servername>/webtest/dataset/`\ `<datasetId>`.
 
 -  **url** goes in omeroweb/omero_webtest/urls.py This maps the URL
    'webtest/dataset/<datasetId>/' to the View function
```

Comment:
I think I can remove/omit comments about OMERO 4.3 or 4.4 since I don't think many users now will be upgrading from 4.x.